lib/internal/Magento/Framework/Filesystem/Driver/File.php

```diff
@@ -1074,8 +1074,8 @@ public function getRealPathSafety($path)
             $path
         );
 
-        if (strpos($path, DIRECTORY_SEPARATOR . '.') === false) {
-            return rtrim($path, DIRECTORY_SEPARATOR);
+        if (!$path || strpos($path, DIRECTORY_SEPARATOR . '.') === false) {
```

Comment:
please add
```suggestion
        if (is_null($path)) {
             return '';
        }
```
at the beginning of the method instead

Reply:
thanks, done